anyric/super-record
# #167133540 Add a Landing Page

#### What does this PR do?
This PR add a landing page for the application.

#### Description of Task to be completed?
- Add a landing page for the application

#### How should this be manually tested?
- Clone this branch
- In the root directory run `python manage.py runserver`
- Navigate to `http://127.0.0.1:8000/` to see the landing page

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167133540](https://www.pivotaltracker.com/story/show/167133540)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
